src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md

```diff
@@ -221,63 +181,26 @@ export default function() {
 
 </div>
 
-## Using checks
-
-Maybe we want to verify that an HTTP transaction worked and that the transaction time stayed below some acceptable value. We can use the [check()](/javascript-api/k6/check-val-sets-tags) function for this:
-
-<div class="code-group" data-props='{"labels": ["stages.js"], "lineNumbers": [true]}'>
+## Running cloud tests
 
-```javascript
-import http from 'k6/http';
-import { check, sleep } from 'k6';
-
-export let options = {
-  vus: 10,
-  duration: '30s',
-};
-
-export default function() {
-  let res = http.get('http://test.k6.io');
-  check(res, {
-    'status was 200': r => r.status == 200,
-    'transaction time OK': r => r.timings.duration < 200,
-  });
-  sleep(1);
-}
-```
+One of the goals with k6 is to support three execution modes to run your k6 tests:
 
-</div>
+- [Local](#running-local-tests): on your local machine or a CI server.
+- [Cloud](/cloud): on cloud infrastructure managed by `k6 Cloud`.
+- Clustered: on more than one machine managed by you. Not supported yet.
 
-_Using check() to verify that transactions work and are fast enough_
+The goal of k6 is to support running a test on the three execution modes without making modifications to the script. 
 
-The above will generate a couple of extra output lines after the test, telling you if your check conditions succeeded or failed during the test. If the check conditions never failed, you will see this:
+For running cloud tests, you must sign up with the k6 Cloud and log in with the CLI. Then, you'd only have to pass your existing script to the `k6 cloud` command.
```

Comment:
I'd rephrase this sentence a bit to clarify: "For running cloud tests from the CLI, you must first register a k6 Cloud account and then log into your account via the CLI. Then, you only have to pass your existing script to the `k6 cloud` command."

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720